hawkbit-device-simulator/src/test/java/org/eclipse/hawkbit/simulator/AllowAllWebSecurityTest.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2019 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.simulator;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import io.qameta.allure.Description;
+import org.eclipse.hawkbit.simulator.http.AuthProperties;
+import org.junit.Test;
+import org.springframework.test.context.TestPropertySource;
+
+@TestPropertySource(properties = {AuthProperties.CONFIGURATION_PREFIX + ".enabled = " + "false"})
```

Comment:
No need to set `AuthProperties.CONFIGURATION_PREFIX + ".enabled = " + "false"` here as it is already set to false in parent class

Reply:
Or remove this property from the parent class

Reply:
In the parent class the **Amqp**Properties are set to false. So this is not redundant

Reply:
okay misunderstood